package.json

```diff
@@ -118,11 +118,11 @@
     "pre-commit": "^1.2.2",
     "prettier": "^1.16.4",
     "pretty-quick": "^1.8.0",
-    "react": "^16.12.0",
-    "react-dev-utils": "^9.0.3",
-    "react-dom": "^16.12.0",
-    "react-is": "16.12.0",
-    "react-test-renderer": "^16.12.0",
+    "react": "^16.12.0 || ^17.0.1",
+    "react-dev-utils": "^11.0.2",
+    "react-dom": "^16.12.0 || ^17.0.1",
+    "react-is": "^16.12.0 || ^17.0.1",
+    "react-test-renderer": "^16.12.0 || ^17.0.1",
```

Comment:
Let's remove the support for React 16, we should support it only for peer dependency.

Reply:
Done in latest commit